api/python/provisioner/inputs.py

```diff
@@ -492,23 +560,40 @@ class StorageEnclosureParams():
 
 
 @attr.s(auto_attribs=True)
-class StorageEnclosure(ParamGroupInputBase):
-    controller_a_ip: str = StorageEnclosureParams.primary_mc_ip
-    controller_b_ip: str = StorageEnclosureParams.secondary_mc_ip
-    controller_user: str = StorageEnclosureParams.controller_user
-    controller_secret: str = StorageEnclosureParams.controller_secret
+class StorageEnclosureDefault(ParamGroupInputBase):
```

Comment:
@divya-vijay the same idea here: please adjust according to the comment above for node attributes

Reply:
`StorageEnclosureParams` addressed all params, so removed `Default` from name. 
Just created a class `StorageNodeParams` as placeholder for node-specific storage params in future.